### DISCUSSION TITLE
Question regarding the plug-in directory structure

### DISCUSSION THREAD

**Discussion by xfzv** [tomasklaen/uosc#622] [Q&A] [answered]
I've been using the [main branch zip](https://github.com/tomasklaen/uosc/archive/refs/heads/main.zip) for as long as I remember and it seems to work just fine.

However, I noticed that the directory structure of the releases archives and the main branch zip differs:

<details>
  <summary>Release archive:</summary>

```console
.
└── scripts
    ├── uosc.lua
    └── uosc_shared
        ├── elements
        ├── lib
        └── main.lua (empty, used for compatibility)
```
</details>

<details>
  <summary>Main branch:</summary>
  
```console
.
└── scripts
    └── uosc
        ├── elements
        ├── lib
        └── main.lua
```
</details>

What is the proper way to use the main branch zip content? Can I keep using it as is?

**Comment by christoph-heinrich** [comment, marked as answer]
That was changed in ade1d1f31962a5c978f119cbab14f367d00381c9.
You can use the main branch as is.

**Reply by xfzv** [reply]
Indeed, I didn't remember this commit...

Thanks!
